controller/resource_test.go

```diff
@@ -147,6 +147,7 @@ func TestServiceStateLister_List(t *testing.T) {
 	}
 
 	for name, c := range cases {
+		c := c
```

Comment:
Is this required?

Reply:
Yes, though it's because I recently added `t.Parallel()` to tests in this file (which reduced the test time for this file from ~60s to ~10s on my laptop).

The reason is this gotcha with how Golang closures capture variables (e.g. https://blog.cloudflare.com/a-go-gotcha-when-closures-and-goroutines-collide/). In some cases these tests modify stuff on `c`, and that's not okay when these run in parallel since they actually share a reference to `c` when they run.

Reply:
Makes sense. What do you think about adding a comment at least the first time?

Reply:
Yep. There _could_ be other issues with enabling those tests to run in parallel. I'm monitoring it, and if it causes more instability I can revert back to no parallelism there.

Reply:
I'd vote for no parallelism and just eat the 60s because guaranteed someone will miss this.